### DISCUSSION TITLE
Any plans to add webhooks support?

### DISCUSSION THREAD

**Discussion by kawpii** [daptin/daptin#127] [Ideas]
As headless CMS webhook support is necessary to rebuild the website when content changes. I see there is websocket support, which seems to offer something similar. But problem I do not think JAMStack platforms(Netlify, CF pages) support them, but they do support webhooks>

**Comment by artpar** [comment]
Hey @kaushalyap, daptin does have websockets and you can subscribe to various kind of events over websockets like you mentioned.

web hooks (incoming requests to trigger changes) are also supported by daptin, you can create actions and define all kind of changes it should do (and the event changes will be sent over to websockets as well)

I am not sure if i understand your request here correctly, can you give a more concrete example in terms of what you think is missing?

**Reply by kawpii** [reply]
What I meant : 
https://developers.cloudflare.com/pages/platform/deploy-hooks
https://www.netlify.com/blog/2020/04/24/automate-contentful-deploys-with-netlify-webhooks/
